### PR TITLE
Pd 157 add page for pxctl sched policy

### PR DESCRIPTION
**Pull request by andreipope**
@harsh-px please review when you have some time.

### CODE REVIEW

**Comment by andreipope** [comment]
@harsh-px please review

File: content/reference/CLI/snapshots.md [hunk 1]
```diff
@@ -145,145 +145,13 @@ Only detached snapshots can be deleted.
...
-
-
-### Deleting a schedule policy
-
-To delete a schedule policy, run `pxctl sched-policy delete` with the policy's name as a parameter:
-
-```text
-pxctl sched-policy delete p1
-```
+For more information on how to create, list, update, and delete your snapshot schedule policies with `pxct`, see [this page](/reference/cli/sched-policy).
```
Comment (nathan-px) [inline content/reference/CLI/snapshots.md]:
```suggestion
For more information on how to create, list, update, and delete your snapshot schedule policies with `pxctl`, see [this page](/reference/cli/sched-policy).
```

File: content/reference/CLI/shared/sched-policy.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+title: Shared content for managing snapshot schedule policies using pxctl
+keywords: portworx, pxctl, snapshot, reference, schedule, policy
+description: Shared content for managing snapshot schedule policies using pxctl
+hidden: true
+---
+
+As an example, to create a policy named `p1` with:
+
+- Periodic backups every 60 minutes (keep the last 5 backups) and
+- Weekly backups every Sunday at 12:00 (keep the last 4 backups),
```
Comment (nathan-px) [inline content/reference/CLI/shared/sched-policy.md]:
```suggestion
- Weekly backups every Sunday at 12:00 (keep the last 4 backups)
```

File: content/reference/CLI/shared/sched-policy.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+title: Shared content for managing snapshot schedule policies using pxctl
+keywords: portworx, pxctl, snapshot, reference, schedule, policy
+description: Shared content for managing snapshot schedule policies using pxctl
+hidden: true
+---
+
+As an example, to create a policy named `p1` with:
+
+- Periodic backups every 60 minutes (keep the last 5 backups) and
```
Comment (nathan-px) [inline content/reference/CLI/shared/sched-policy.md]:
```suggestion
- Periodic backups every 60 minutes (keep the last 5 backups)
```

File: content/reference/CLI/shared/sched-policy.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+---
+title: Shared content for managing snapshot schedule policies using pxctl
+keywords: portworx, pxctl, snapshot, reference, schedule, policy
+description: Shared content for managing snapshot schedule policies using pxctl
+hidden: true
+---
+
+As an example, to create a policy named `p1` with:
```
Comment (nathan-px) [inline content/reference/CLI/shared/sched-policy.md]: We can't just split a sentence 
```suggestion
The following example creates a policy named `p1` with the following properties:
```

File: content/reference/CLI/shared/sched-policy.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+- Periodic backups every 60 minutes (keep the last 5 backups) and
+- Weekly backups every Sunday at 12:00 (keep the last 4 backups),
+
+run the following command:
```
Comment (nathan-px) [inline content/reference/CLI/shared/sched-policy.md]:
```suggestion
Run the following command to create the `p1` backup policy:
```

File: content/reference/CLI/shared/sched-policy.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+```text
```
Comment (nathan-px) [inline content/reference/CLI/shared/sched-policy.md]: indent these to sit under the steps

File: content/reference/CLI/shared/sched-policy.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+pxctl sched-policy create --periodic 60,5 --weekly sunday@12:00,4 p1
+```
+
+You can add schedule policies either when a volume gets created or afterward.
```
Comment (nathan-px) [inline content/reference/CLI/shared/sched-policy.md]: If these are all steps within a single task, we need to reformat and rewrite them into steps. 

If they're distinct tasks containing only one step each, they need headings describing and separating them. 

If they're not steps that we want readers to perform, we should change the wording to remove the "run the following command" instructions and make it purely descriptive.

**Comment by andreipope** [comment]
> I have some changes for this, mostly on the writing, since it predates the style guide.

Yup, this PR  is sitting here for a long time now. I'll revisit it.

File: content/reference/CLI/shared/sched-policy.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+Here is an example of how  you can add a schedule policy when the volume is created:
+
+```text
+pxctl volume create --policy p1 vol1
+```
+
+To add a schedule policy later, enter the following command:
```
Comment (sangleganesh) [inline content/reference/CLI/shared/sched-policy.md]: add -> add/update

Comment (sangleganesh) [reply content/reference/CLI/shared/sched-policy.md]: to remove policy from volume set snap interval to 0.

pxctl volume snap-interval-update --periodic 0 vol1

**Comment by nathan-px** [comment]
@andreipope I spoke with Ganesh -- once you address these last 2 comments, you can merge this.

File: content/reference/CLI/shared/sched-policy.md (hunk 8 above)
Comment (andreipope) [reply content/reference/CLI/shared/sched-policy.md]: @sangleganesh Thanks for taking the time to review the PR and for suggestions. I just updated it now.

File: content/reference/CLI/shared/sched-policy.md (hunk 7 above)
Comment (andreipope) [reply content/reference/CLI/shared/sched-policy.md]: Thanks, I appreciate the tip! I've just made the changes.

**Pull request merged**